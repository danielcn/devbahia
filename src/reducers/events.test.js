import event from './events';

describe('Event Snapshot', () => {
  test('events', () => {
    expect(event).toMatchSnapshot();
  });
});
