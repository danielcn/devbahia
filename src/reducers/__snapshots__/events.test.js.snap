// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Snapshot events 1`] = `[Function]`;
